examples/slack-connector/pom.xml

```diff
@@ -0,0 +1,92 @@
+
+  <groupId>com.salesforce.chatbot</groupId>
+  <artifactId>slack-demo-app-connector</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <name>slack-demo-app-connector</name>
+
+  <properties>
+    <java.version>1.8</java.version>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <bootstrap.version>4.2.1</bootstrap.version>
+    <slack-api-version>1.12.1</slack-api-version>
+    <spring-reactor-core-version>3.4.5</spring-reactor-core-version>
+  </properties>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.5.6</version> <!-- lookup parent from repository -->
+  </parent>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.slack.api</groupId>
+      <artifactId>slack-api-client</artifactId>
+      <version>${slack-api-version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.slack.api</groupId>
+      <artifactId>bolt</artifactId>
+      <version>${slack-api-version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.slack.api</groupId>
+      <artifactId>bolt-servlet</artifactId>
+      <version>${slack-api-version}</version>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-handler</artifactId>
+      <version>4.1.63.Final</version>
+    </dependency>
+    <dependency>
+      <groupId>io.projectreactor</groupId>
+      <artifactId>reactor-core</artifactId>
+      <version>${spring-reactor-core-version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.salesforce.einsteinbot</groupId>
+      <artifactId>einstein-bot-channel-connector-java-starter</artifactId>
+      <version>2.0.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.newrelic.telemetry</groupId>
+      <artifactId>micrometer-registry-new-relic</artifactId>
+      <version>0.8.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-thymeleaf</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.webjars</groupId>
+      <artifactId>salesforce-lightning-design-system</artifactId>
+      <version>2.13.3</version>
+    </dependency>
+  </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-maven-plugin</artifactId>
+        <version>2.4.1</version>
+      </plugin>
+      <plugin>
+        <groupId>com.heroku.sdk</groupId>
+        <artifactId>heroku-maven-plugin</artifactId>
+        <version>3.0.3</version>
+        <configuration>
+          <processTypes>
+            <web>java -Dserver.port=$PORT $JAVA_OPTS -cp target/classes:target/dependency/* com.salesforce.connector.Application</web>
+          </processTypes>
+          <appName>glacial-dawn-73028</appName>
+        </configuration>
+      </plugin>
```

Comment:
Probably can skip this too - we cannot include app name here because it will be specific to each deployment. We can just add this as instructions in README.

Reply:
Replaced appName with placeholder but kept the configuration element. Instructions in README for how to update it.